wiki/Tournaments/GBC/GBC_2023_Spring/en.md

```diff
@@ -0,0 +1,207 @@
+   - Custom skin elements **must not** be used to alter core gameplay mechanics.
+9. Possible mod choices for the FreeMod bracket are **No Fail, **Hidden**, **Fade In**, **Flashlight**, and **Mirror**.
+10. The referees will select the beatmaps in order to play the match. If a player encounters a technical problem during a match, players may request a rematch from the referee after all the beatmaps have been selected.
+    - "Lag spikes" will not be considered as a valid reason to rematch a beatmap.
+    - The rematch should happen after completing scores for the other beatmaps in the mappool.
+    - This rule should not to be abused. Referees may veto a rematch request if they find that this is the case.
+11. Unless there are special reasons, the organiser will only record each player's first score of the beatmap in the match room.
+12. The default rest time is **120 seconds**. If all players in the room agree to start early and get ready, the referee may start the match early.
+13. Players are expected to keep the match running fluently and without delays. Excessive delays from the players' side may result in penalties being applied at the referee's discretion. Disrupting the match by foul play, insulting or provoking other players or staff, delaying the match, or other deliberate inappropriate misbehaviour are strictly prohibited, and will be punished accordingly.
+14. All players and staff must be treated with respect. Instructions from referees and the host are to be followed. Decisions labelled as final are not to be objected.
+15. The multiplayer chat rooms underlie the [osu! community rules](/wiki/Rules). All chat rules apply to the multiplayer chat rooms, too.
+16. Disrupting the match by foul play or seed manipulation will result in disqualification, and any such players will be announced.
+17. Unexpected incidents will be handled by the host. Referees may allow higher tolerance depending on the given circumstances. This is up to their discretion.
+18. The host may request liveplays or recordings of individual players at any point.
+19. The host reserves the right to modify these rules at any moment. Any such changes will be announced in advance.
+
+### Registration
+
+1. Please **fill [this form](https://wj.qq.com/s2/11825634/aba7/)** to register.
+   - Player **must** join [this QQ group](https://jq.qq.com/?_wv=1027&k=zqBeTqMW) after register.
+   - If you cannot use QQ, please use osu! or Discord to contact the organiser and use Discord to communicate.
+2. To ensure valid and serious registrations, every registered user will be manually checked by the host.
+   - **There are no rank limitations for this tournament.**
+   - To be successfully registered, a player must not have violated any [osu! community rules](/wiki/Rules) within the last 12 months.
+   - When registering, the organiser will refer to the player's play counts; heavy derankers and smurfs may be forced to upgrade.
```

Comment:
what does "upgrading" mean

Reply:
For heavy derankers and smurfs, after checking by the organizer and staff, if the organizer and staff believe that a player meets the heavy derankers and smurfs situation, even if his PP will make him to be placed in a relatively weak group, we will manually intervene to promote the player to a higher level group that matches the player's actual level.